corehq/apps/api/tests/test_auth.py

```diff
@@ -85,6 +85,8 @@ def test_login_no_auth(self):
 
     def test_login_with_api_key_auth(self):
         self.assertAuthenticationSuccess(LoginAuthentication(), self._get_request_with_api_key())
+        self.api_key.refresh_from_db()
+        self.assertIsNotNone(self.api_key.last_used)
```

Comment:
I was not sure if there is a good way to include that in `assertAuthenticationSuccess`?

Reply:
the way you have it here seems appropriate to me